### ISSUE TITLE
"Cursor wouldlike to access data from other apps" endless loop

### ISSUE THREAD

**Issue by Stooovie** [getcursor/cursor#1429]
Sonoma 14.4.1, Cursor 0.31.3. Endless loop of this, clicking Allow forever doesn't fix:

![Screenshot 2024-05-03 at 17 46 11](https://github.com/getcursor/cursor/assets/10572373/af207116-ef71-4580-b4ad-dc3c85801aa6)

Any tips?

**Comment by surim0n** [comment]
i have clicked both dozens of times and it will not go away. ive restarted everything.

**Comment by philliphartin** [comment]
This has recently started driving me mad

**Comment by josh-may** [comment]
Same issue!

**Comment by JackBlair87** [comment]
Same issue!

**Comment by urbanonymous** [comment]
Same issue...

**Comment by glycerine** [comment]
Yep. Endless loop here too, with the most recent Cursor. Sad. I may have
to unsubscribe since it has become unusable (and requires a machine shutdown whenever it goes into the endless loop of asking for permissions.)

On Mac OS Sonoma 14.0.
